src/js/components/InstitutionNameAndId.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const InstitutionNameAndId = ({ name, id }) => {
```

Comment:
This component feels a bit unnecessary, like it could have just been a `renderNameAndId` function in the Institution component.

That said, I think stylistically you were going for a super simple institution component that just passes through data and doesn't handle render logic outside of organizing the proper components, so I'm okay with it. Also, the nameAndId component could grow/change and that would be masked in this approach.

Reply:
Exactly, I would like us to do more of this ... breaking down components into smaller, "dumber" components. It comes back to the ["Thinking in React"](https://facebook.github.io/react/docs/thinking-in-react.html) article. Over time we've moved away from that and IMO it would be good to start moving back towards it.

This is also why I think we will, eventually, need to think about how are [`/src` is structured](https://medium.com/styled-components/component-folder-pattern-ee42df37ec68). By breaking things out file names become _too important_, having them in a directory helps a lot.

Just as an example (might not be necessary in this case but could be done), notice the `Institution` prefix already on the new components. And if we break out `InstitutionPreviousSubmissions` anymore you end up with things like `InstitutionPreviousSubmissionsButton` and `InstitutionPreviousSubmissionsList`, etc.